tests/test_citations.py

```diff
@@ -245,6 +254,39 @@ def test_citation_to_citeproc_pubmed_book():
         citation_to_citeproc('pmid:29227604')
 
 
+def test_citation_to_citeproc_isbn():
+    csl_item = citation_to_citeproc('isbn:9780387950693')
+    assert csl_item['type'] == 'book'
+    assert csl_item['title'] == 'Complex Analysis'
+
+
+def test_citation_to_citeproc_isbn_title_with_quotation_mark():
+    csl_item = citation_to_citeproc('isbn:9780312353780')
+    assert csl_item['type'] == 'book'
+    assert csl_item['title'].startswith('"F" is for Fugitive')
+
+
+def test_get_isbn_citeproc_citoid_weird_date():
+    """
+    isbn:9780719561023 has a date value of "(2004 printing)"
+    https://en.wikipedia.org/api/rest_v1/data/citation/mediawiki/9780719561023
```

Comment:
Note that the author is also weird for this book.  Manubot produces:
```
    "author": [
      {
        "given": "1969-",
        "family": "Frey, James,"
      }
    ],
```

Reply:
Yes authors are quite bad. In almost all cases I tried with Citoid, the author metadata was imperfect.